### DISCUSSION TITLE
v0.29.0

### DISCUSSION THREAD

**Discussion by WillTheDeveloper** [WillTheDeveloper/StudyPortal#107] [Announcements]
## What's Changed
* Assignment emails by @WillTheDeveloper in https://github.com/WillTheDeveloper/StudyPortal/pull/105

**Full Changelog**: https://github.com/WillTheDeveloper/StudyPortal/compare/v0.28.1...v0.29.0

<hr /><em>This discussion was created from the release <a href='https://github.com/WillTheDeveloper/StudyPortal/releases/tag/v0.29.0'>v0.29.0</a>.</em>

**Comment by mr-cheffy** [comment]
nice

**Reply by hrszpuk** [reply]
based

**Reply by WillTheDeveloper** [reply]
based
